app/konnect/updates.md

```diff
@@ -3,11 +3,115 @@ title: Konnect Cloud Updates
+
+**{{site.konnect_short_name}} UI/UX redesign**
+: The {{site.konnect_short_name}} UI now has reworked navigation, refreshed colors and styles, and a focus on consistent interactions throughout the application.
+
+: Navigation redesign details:
+  * Services is renamed to Service Hub
+  * Runtimes is renamed to Runtime Manager
+  * Shared Config is now part of Runtime Manager
+  * Dev Portal settings have moved from the general settings page to the Dev Portal section
+  * Refreshed iconography for the main menu
+
+**Dev Portal default theme**
+: The Dev Portal now comes with a new default theme: "Mint Rocket".
+
+**Dev Portal SSO support with your own IdP**
+: You can now configure custom identity providers (IdPs) for the Dev Portal.
+: See the [SSO documentation](/konnect/dev-portal/customization/#single-sign-on) for more information.
+
+**Simplified custom Dev Portal URL setup**
+: SSL certificate generation for custom URLs is now handled by {{site.konnect_short_name}}.
+
+: To set up a custom URL, enter your desired custom portal URL in the Dev Portal settings (for example, `portal.example.com`) and set up a CNAME record to the {{site.konnect_short_name}} organization's Dev Portal domain. See the [Dev Portal customization doc](/konnect/dev-portal/customization) for more information.
+
+**Vitals metrics and dashboards**
+: The Vitals overview dashboard now provides metrics for services cataloged by Service Hub within a selected time interval.
+You can view a graph for each category by clicking **Traffic** or **Errors**, and switching between the two views. Each graph is filterable by time frame.
+
+: In the Service Hub, graphs for services and routes now show data up to the last 30 days.
+
+**Vitals custom reports** <span class="badge alpha"></span>
+
+: Through the Vitals menu, you can create custom reports to track API calls based on services, routes, or applications.
+
+: See the [custom reports documentation](/konnect/vitals/generate-reports) for more information.
+
+**Custom plugin requirements have changed**
+: Some custom plugin limitations have changed or been removed.
+See the latest requirements in the [plugin doc](/konnect/servicehub/plugins).
+
+: Custom plugins can't be added directly through the {{site.konnect_saas}} application.
+If you have a custom plugin you want to use in {{site.konnect_short_name}}, contact [Kong Support](https://support.konghq.com/).
+
+**Refactored documentation**
+: The {{site.konnect_short_name}} documentation has been refactored to match the new {{site.konnect_short_name}} navigation, and generally reworked to provide a better experience for {{site.konnect_short_name}} users.
+
+**Known issues/limitations**
+: The `cloud.konghq.com` environment has the following restrictions:
+  * Dev Portal and application registration can only be used with the default runtime group.
```

Comment:
Specifically this should be "Application registration through Dev Portal can only be used with the default runtime group. This restriction will be removed in a future update to Konnect".